vmdb/app/models/orchestration_template.rb

```diff
@@ -90,6 +93,16 @@ def validate_content(manager = nil)
     "No #{ui_lookup(:model => 'ExtManagementSystem').downcase} is capable to validate the template"
   end
 
+  def validate_format
+    "Cannot validate format because template type is unknown"
```

Comment:
Oh, so is the intention that this would never actually be called on a base class?  If so, maybe it's better to have the base class' method just `raise NotImplementedError` as in [vm_or_template.rb](https://github.com/bzwei/manageiq/blob/fefb52bfca5cf5dcad3d4544571e465ab6b003a0/vmdb/app/models/vm_or_template.rb#L1285).

What do you think?